crates/bevy_macro_utils/src/lib.rs

```diff
@@ -78,6 +78,22 @@ impl BevyManifest {
         deps.and_then(find_in_deps)
             .or_else(|| deps_dev.and_then(find_in_deps))
     }
+
+    /// Returns the path for the crate with the given name.
+    ///
+    /// This is a convenience method for constructing a [manifest] and
+    /// calling the [`get_path`] method.
+    ///
+    /// This method should only be used where you just need the path and can't
+    /// cache the [manifest]. If caching is possible, it's recommended to create
+    /// the [manifest] yourself and use the [`get_path`] method.
+    ///
+    /// [`get_path`]: Self::get_path
+    /// [manifest]: Self
+    pub fn just_get_path(name: &str) -> syn::Path {
```

Comment:
```suggestion
    pub fn simple_path(name: &str) -> syn::Path {
```

Reply:
I feel like simple implies that the path is somehow different from using `get_path` as normal. What about `get_path_direct`?
 
Alternatively, I could maybe add an extension trait for `Path` that gives it like a `Path::from_bevy_path(...)` or something.

Reply:
Done!